vagrant/scripts/install_iml_local_agent.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+yum install -y yum-plugin-copr
+yum copr enable -y managerforlustre/manager-for-lustre-devel
+mkdir -p /tmp/agent-rpms
+scp adm:/tmp/agent-rpms/*.rpm /tmp/agent-rpms
+yum localinstall -y /tmp/agent-rpms/*.rpm
```

Comment:
No reason to use `yum localinstall` unless on centos 5. It should still work the same as `yum install`.

Reply:
ref?

Reply:
yeah, guess it makes sense that command is deprecated... I liked that it signals intent.